.travis.yml

```diff
@@ -44,8 +44,27 @@ deploy:
       all_branches: true
       condition: $AWSAccessKeyId
   - provider: script
-    script: "mvn com.github.github:site-maven-plugin:site -N"
+    script: "mvn org.reficio:p2-maven-plugin:site -N"
     skip_cleanup: true
     on:
       branch: ustutt
-      tags: true
```

Comment:
No tags any more?

Reply:
the conditions might be cumulative, meaning both would need to apply, which is not intended.

Reply:
Yeah, indeed. We changed it to `all_branches`...